apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/kotlin/file/OperationBuilder.kt

```diff
@@ -79,6 +80,9 @@ class OperationBuilder(
         .addSuperinterface(superInterfaceType())
         .maybeAddDescription(operation.description)
         .makeDataClass(operation.variables.map { it.toNamedType().toParameterSpec(context) })
+        .applyIf(useJvmAnnotations) {
```

Comment:
Also curious to hear if there's a better way to know if this annotation would even be useful (there are arguments that have default values) 🤔

Reply:
I think that's a good idea. Keeps the generated code more readable. It'd be confusing to see a `@JvmOverloads`
 annotation on a function without any default parameter.